packages/fhir-converter/src/templates/cda/Resources/MedicationStatement.hbs

```diff
@@ -45,13 +45,17 @@
                 "route":{{>DataType/CodeableConcept.hbs code=medicationStatement.routeCode}},
                 "doseAndRate": [
                     {
-                        {{#with (extractRangeFromQuantity medicationStatement.doseQuantity) as |rangeObj|}}
-                            {{#if rangeObj.isValid}}
-                                "doseRange": {{>DataType/Range.hbs range=rangeObj.range}},
-                            {{else}}
-                                "doseQuantity": {{>DataType/Quantity.hbs quantity=../medicationStatement.doseQuantity}},
-                            {{/if}}
-                        {{/with}}
+                        {{#if medicationStatement.doseQuantity.low}}
+                            "doseRange": {{>DataType/Range.hbs range=medicationStatement.doseQuantity}},
+                        {{else}}
+                            {{#with (extractRangeFromQuantity medicationStatement.doseQuantity) as |rangeObj|}}
+                                {{#if rangeObj.isValid}}
+                                    "doseRange": {{>DataType/Range.hbs range=rangeObj.range}},
+                                {{else}}
+                                    "doseQuantity": {{>DataType/Quantity.hbs quantity=../medicationStatement.doseQuantity}},
+                                {{/if}}
```

Comment:
<img width="778" alt="Screenshot 2024-09-16 at 22 00 21" src="https://github.com/user-attachments/assets/e545bbc5-b057-4761-8b74-c4c616e48ac6">

missing all doses that look like this :(

Reply:
💪